frame_infer_log_top_path: "/home/bnn_attitude_predictor_with_image/test2/infer_log/frame_infer"
frame_infer_log_file_name: "frame_infer_result.csv"

saved_log_csv_top_path: "/home/bnn_attitude_predictor_with_image/test2/infer_log/eval_frame_infer"
saved_log_csv_file_name: "eval_result.csv"

dataset_data_top_path: "/home/ssd_dir/dataset_image_to_gravity_ozaki/AirSim/1cam/Neighborhood_40deg_1000samples"
dataset_data_file_name: "imu_camera.csv"

loop_period: 3 #wait sec